readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template Guide
+
+Read this document before you start working on the README template.
+
+
+*Note: As your project evolves and grows, so should your README file. READMEs should be revisited regularly to ensure relevant information is captured for new users.*
```

Comment:
I would write this in a more general way (and avoid pronouns): 
As projects evolve and grow, so should their according README files.

Reply:
Thanks for providing this comment! After discussion, we will keep the pronouns in alignment with the Google Dev style guide recommending second-person use so readers feel a stronger connection with the template and guide.